nautobot/dcim/api/serializers.py

```diff
@@ -705,8 +707,6 @@ class Meta:
 
 class DeviceRoleSerializer(NautobotModelSerializer):
     url = serializers.HyperlinkedIdentityField(view_name="dcim-api:devicerole-detail")
-    device_count = serializers.IntegerField(read_only=True)
-    virtualmachine_count = serializers.IntegerField(read_only=True)
```

Comment:
Is this a removal of existing functionality? Should be documented in the changelog and in the migration docs if so.

...Actually, shouldn't this serializer go away entirely with the removal of `DeviceRole` as a distinct model?

Reply:
Every thing relating to `DeviceRole` and `ipam.Role` would all be removed